vite.config.mts

```diff
@@ -0,0 +1,46 @@
+import react from '@vitejs/plugin-react-swc';
```

Comment:
Is there a separate config file for the vite production build ?

Reply:
There is not, currently as it isn't necessary. Do you have anything that needs changing in the production build?

Reply:
Not really. I was just asking as I was trying to draw parallels to webpack.